getting-started-reactive-crud/README.md

```diff
@@ -0,0 +1,117 @@
+# Getting started with Quarkus
+
+This is a CRUD service exposing endpoints over REST. These endpoints are used for get, add, update
```

Comment:
Don't we have a guide for that?

Because we usually avoid duplicating content in the quickstarts. Some of them have a README with a link to the guide.